### ISSUE TITLE
Fixed price / React / Ruby sample quite broken.

### ISSUE THREAD

**Issue by iHiD** [stripe-samples/subscription-use-cases#126]
Instructions on website are: 
```bash
cd subscription-use-cases/server
bundle install
ruby server.rb
```

However, `server.rb` is in a subdirectory:

<img width="803" alt="Screenshot 2021-07-08 at 15 33 57" src="https://user-images.githubusercontent.com/286476/124940462-eb2b3a80-e001-11eb-8bf3-210fc92f6862.png">

So running that gives this:
```
➜ ruby server.rb
Traceback (most recent call last):
ruby: No such file or directory -- server.rb (LoadError)
```

If I cd into `server` and run `ruby server.rb`, I get this:
<img width="1435" alt="Screenshot 2021-07-08 at 15 34 31" src="https://user-images.githubusercontent.com/286476/124940552-ff6f3780-e001-11eb-873b-138181e39ba9.png">

Here's a loom for you to watch it all happening:
https://www.loom.com/share/11ebb02ed1dc401d8a17f474e404a9a3

**Comment by cjavilla-stripe** [comment]
Thanks for the loom video and the report @iHiD! That failing check was for the html client which isn't compatible with the react client. I'm removing that check now. Sorry for any confusion about where to run the server. We have a Gemfile in the root, but that's for running the rspec tests across all server languages. The correct way is the last you tried by running `ruby server.rb` from inside the `/server` directory.

**Comment by iHiD** [comment]
Thanks for the fast response 🙂

**Comment by cjavilla-stripe** [comment]
I've been testing this a bit and there's some issue with proxying for create-react-app to sinatra (might be CORS?). Working on a fix now. Thanks for your patience, @iHiD! :)

**Comment by cjavilla-stripe** [comment]
Turns out it was related to webpack dev server settings used by create-react-app's proxy while using `localhost` instead of binding to 0.0.0.0. Fixed in 692fea0 

Can you give it another shot and close if that works?

**Comment by iHiD** [comment]
Yep. Much better. Thanks!
